web/content/docs/benchmarks/reactive-transport/kineticreactant_allascomponents/KineticReactant2.pandoc

```diff
@@ -0,0 +1,92 @@
++++
+author = "Johannes Boog"
+weight = 143
+project = "Parabolic/ReactiveTransport/KineticReactant_AllAsComponents/"
+date = "2010-06-25T14:41:09+01:00"
+title = "Solute transport including kinetic reaction"
+
+[menu]
+  [menu.benchmarks]
+    parent = "Reactive transport"
+
++++
+
+{{< data-link >}}
+
+
+
+
+## Overview
+
+This scenario describes the transport of two solutes (Snythetica and Syntheticb) through a saturated media.
+Both solutes react to Productc according to $\text{Productd}=\text{Synthetica}+0.5~\text{Syntheticb}$.
+The speed of the reaction is described with a first--order relationship $\frac{dc}{dt}=U(\frac{c_{\text{Synthetica}}}{K_m+c_{\text{Syntheticb}}})$.
+The coupling of OGS-6 and IPhreeqc used for simulation requires to simulate the transport of H^+^--ions, additionally.
+This is required to adjust the compulsory charge balance computation executed by Phreeqc.
+The solution by OGS-6--IPhreeqc will be compared to the solution by a coupling of OGS-5--IPhreeqc.
+
+
+## Problem Description
+
+**1d scenario:**
+The 1d--model domain is 0.5 m long and discretized into 200 line elements.
+The domain is saturated at start--up ($p(t=0)=1.0e^{-5}~\text{Pa}$).
```

Comment:
`1e^x` is ambiguous. You probably mean `1^x` or `1ex`. I prefer the former.

Reply:
Updated